faunadb/query.py

```diff
@@ -40,6 +43,43 @@ def quote(expr):
   """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
   return {"quote": expr}
 
+thread_local = local()
+thread_local.lambda_auto_var_number = 0
```

Comment:
What happens if we import this than create a thread that calls this? Will it be available to that thread? If not this won't be thread safe at all.